eodatasets3/prepare/sentinel_L1C_prepare.py

```diff
@@ -0,0 +1,393 @@
+"""
```

Comment:
Keep py file names lowercase, for consistency. 

(I'm surprised the pep8 linter doesn't complain about this)